portfolio/src/main/java/com/google/sps/data/Post.java

```diff
@@ -5,11 +5,18 @@
 public class Post {
   private String title;
   private String content;
+  private long id;
 
-  /** Returns whether this game has ended. */
   public Post(String title, String content) {
     this.title = title;
```

Comment:
nit.
you can replace the contents of this constructor here with
```
this(title, content, uniqueIdentifierGenerator());
```
and save a couple lines

Reply:
Nice.